README.md

```diff
@@ -155,6 +162,8 @@ rails generate decorator Article
 
 ...to create the `ArticleDecorator`.
 
+Generated decorators will inherit from `Draper::Decorator` unless an `ApplicationDecorator` exists.
```

Comment:
I think it might be more in context to move this up with the rest of the new documentation. Something like:

> To create an `ApplicationController` that all generated decorators inherit from, run:
> 
> ```
> rails generate draper:install
>````

I don't think the additional comment about them inheriting from `Draper::Decorator` otherwise is necessary given that the paragraph above it says that all decorators inherit from `Draper::Decorator`.

I could also see this documentation living in the "Installation" section of the README at some point. I think we should leave it here for now, but just something to think about.